### PR TITLE
reduce property record storage on JIT server

### PR DESCRIPTION
**Pull request by MikeHolman**
We only really need to know if a property ID is numeric, rest of property record was only being used for debug output.

### CODE REVIEW

File: lib/JITIDL/JITTypes.h [hunk 1]
```diff
@@ -804,7 +804,7 @@ typedef struct JITOutputIDL
 typedef struct UpdatedPropertysIDL
 {
     unsigned int reclaimedPropertyCount;
-    unsigned int newRecordCount;
+    unsigned int newPropertyCount;
     [size_is(reclaimedPropertyCount)] int * reclaimedPropertyIdArray;
-    [size_is(newRecordCount)] PropertyRecordIDL ** newRecordArray;
```
Comment (leirocks) [inline lib/JITIDL/JITTypes.h]: >PropertyRecordIDL [](start = 30, length = 17)

remove PropertyRecordIDL definition?

File: lib/Backend/Sym.cpp [hunk 2]
```diff
@@ -1111,12 +1100,17 @@ Sym::Dump(IRDumpFlags flags, const ValueType valueType)
         switch (propertySym->m_fieldKind)
         {
         case PropertyKindData:
-        {
             propertySym->m_stackSym->Dump(flags, valueType);
-            Js::PropertyRecord const* fieldName = propertySym->m_func->GetThreadContextInfo()->GetPropertyRecord(propertySym->m_propertyId);
-            Output::Print(_u("->%s"), fieldName->GetBuffer());
+            if (JITManager::GetJITManager()->IsOOPJITEnabled())
```
Comment (leirocks) [inline lib/Backend/Sym.cpp]: !

**Comment by leirocks** [comment]
:shipit:

**Comment by MikeHolman** [comment]
@dotnet-bot test this please

**Pull request merged**